client/ayon_xstudio/plugins/load/open_in_xstudio.py

```diff
@@ -94,9 +98,7 @@ def load(
             # we output the seq path with hashes.
             idxs = list(seq.indexes)
             pad = "#" * (seq.padding + len(str(idxs[0])))
```

Comment:
```suggestion
            pad = "#" * len(str(idxs[-1]))
```

Alternatively safer is to use this, since some versions of clique was returning on seq.padding `0`. Assuming this might be history - just mentioning that this used to happen to me and this was workaround.